src/votekit/matrices/mentions.py

```diff
@@ -0,0 +1,116 @@
+from ..ballot import Ballot
+from ..pref_profile import PreferenceProfile
+from ._convert_dict_to_mat import _convert_dict_to_matrix
+import numpy as np
+from fractions import Fraction
+
+
+def mention(i: str, ballot: Ballot) -> bool:
+    """
+    Returns true if candidate i is on the ballot, either in the ranking or the scoring.
+    Candidates who receive 0 points are not counted as mentioned.
+
+    Args:
+      i (str): Candidate name.
+      ballot (Ballot): Ballot.
+
+    Returns:
+      bool: True if i appears anywhere in ballot.
+    """
+    cands: set[str] = set()
+    if ballot.scores:
+        cands = cands.union(ballot.scores.keys())
+
+    if ballot.ranking:
+        cands = cands.union(c for s in ballot.ranking for c in s)
+
+    return i in cands
+
+
+def comention(i: str, j: str, ballot: Ballot) -> bool:
+    """
+    Takes candidates i,j and returns true if they both appear on the ballot, either in the ranking
+    or the scoring. Candidates who receive 0 points are not counted as mentioned.
+
+    Args:
+      i (str): Candidate name.
+      j (str): Candidate name.
+      ballot (Ballot): Ballot.
+
+    Returns:
+      bool: True if both i and j appear in ballot.
+    """
+
+    cands: set[str] = set()
+    if ballot.scores:
+        cands = cands.union(ballot.scores.keys())
+
+    if ballot.ranking:
+        cands = cands.union(c for s in ballot.ranking for c in s)
+
+    return i in cands and j in cands
```

Comment:
I think that you can do this in one line

```suggestion
def comention(i: str, j: str, ballot: Ballot) -> bool:
    """
    Takes candidates i,j and returns true if they both appear on the ballot, either in the ranking
    or the scoring. Candidates who receive 0 points are not counted as mentioned.

    Args:
      i (str): Candidate name.
      j (str): Candidate name.
      ballot (Ballot): Ballot.

    Returns:
      bool: True if both i and j appear in ballot.
    """
    return mention(i, ballot) and mention(j, ballot)
```